src/components/calcite-flow-item/calcite-flow-item.tsx

```diff
@@ -76,23 +81,27 @@ export class CalciteFlowItem {
 
   // --------------------------------------------------------------------------
   //
-  //  Private Properties
+  //  Public Methods
   //
   // --------------------------------------------------------------------------
 
-  @Element() el: HTMLCalciteFlowItemElement;
+  /**
+   * Removes the component.
+   */
+  @Method()
+  async back(): Promise<void> {
```

Comment:
From an API-perspective, I would expect the back-handling logic to be on the parent and not the individual items. What's the main reasoning behind the move? Maybe I'm missing something.

Reply:
@jcfranco

1.  I was thinking that each panel may have its own logic for when it is able to go back or not. Like a panel may have saved changes and another not need to worry about it. This way, a user can supply a beforeBack on each flow-item.
2. If we have `beforeBack` on the `flow-item` and `back` on the `flow`, there is a disconnect there. If you just have a `flow-item-panel`, the back button does nothing.
3. The click event on the `flow-item` calls the `back` method. If it were on the `flow`, the `flow-item` would have to get its parent and call back? Kind of related to #2 above.

Thoughts?

Reply:
Will move `back()` back to the `flow` 👍